src/Flex.js

```diff
@@ -4,14 +4,10 @@ import {FLEX} from './constants'
 import theme from './theme'
 import Box from './Box'
 
-const Flex = styled(Box)`
```

Comment:
Could this just be `const Flex = Box` 🤔

Reply:
No, because we set default props on it

Reply:
Oh weird, I'm kind of surprised it doesn't work still 🤔 I'd think that you could create a new constant and then set default props on that and they'd get respected but yeah it looks like it doesn't work!

Reply:
It would be a new constant, but both `Flex` and `Box` are pointing to the same underlying object